packages/main/src/themes/Icon.css

```diff
@@ -28,8 +28,17 @@
 }
 
 .ui5-icon-root {
+    width: 1rem;
 	display:flex;
 	transform: scale(1, -1);
 	transform-origin: center;
 	outline: none;
 }
+
+.ui5-icon-svg-wrapper {
```

Comment:
Overall I would suggest no changes to the icon but a solution from the outside. Can you maybe wrap the icon in something else externally and scale/center it if necessary?

Changing the icon structure may damage functionality everywhere, we should do so as a last resort IMO